code/modules/projectiles/guns/specialist.dm

```diff
@@ -65,51 +78,85 @@
 		return TRUE
 
 /datum/action/item_action/specialist/aimed_shot/proc/use_ability(atom/A)
-	var/mob/living/carbon/human/H = owner
+	var/mob/living/carbon/human/human = owner
 	if(!istype(A, /mob/living))
 		return
 
-	var/mob/living/M = A
+	var/mob/living/target = A
 
-	if(M.stat == DEAD || M == H)
+	if(target.stat == DEAD || target == human)
 		return
 
 	var/obj/item/weapon/gun/rifle/sniper/sniper_rifle = holder_item
 	if(world.time < sniper_rifle.aimed_shot_cooldown)
 		return
 
-	if(!check_can_use(M))
+	if(!check_can_use(target))
 		return
 
 	sniper_rifle.aimed_shot_cooldown = world.time + sniper_rifle.aimed_shot_cooldown_delay
 
 	///Add a decisecond to the default 1.5 seconds for each two tiles to hit.
-	var/distance = round(get_dist(M, H) * 0.5)
+	var/distance = round(get_dist(target, human) * 0.5)
 	var/f_aiming_time = sniper_rifle.aiming_time + distance
-	if(HAS_TRAIT(M, TRAIT_SPOTTER_LAZED))
-		f_aiming_time *= 0.5
-
-	var/image/I = image(icon = 'icons/effects/Targeted.dmi', icon_state = "locking-sniper", dir = get_cardinal_dir(M, H))
-	I.pixel_x = -M.pixel_x + M.base_pixel_x
-	I.pixel_y = (M.icon_size - world.icon_size) * 0.5 - M.pixel_y + M.base_pixel_y
-	M.overlays += I
-	if(H.client)
-		playsound_client(H.client, 'sound/weapons/TargetOn.ogg', H, 50)
-	playsound(M, 'sound/weapons/TargetOn.ogg', 70, FALSE, 8, falloff = 0.4)
-
-	if(!do_after(H, f_aiming_time, INTERRUPT_ALL|BEHAVIOR_IMMOBILE, NO_BUSY_ICON))
-		M.overlays -= I
+
+	var/aiming_buffs
+
+	if(sniper_rifle.enable_aimed_shot_laser)
+		aiming_buffs++
+
+	if(HAS_TRAIT(target, TRAIT_SPOTTER_LAZED))
+		aiming_buffs++
+
+	if(aiming_buffs == 1)
+		f_aiming_time *= 0.6
```

Comment:
Undocumented nerf to spotter . It used to be 0.5 + wicked way to apply the stuff lol

Reply:
stan knows about it

figure out a better way loser!

Reply:
![image](https://user-images.githubusercontent.com/53100513/208750199-6074bcd6-296f-4e96-9350-03735eb2c16c.png)

ok i did that instead